### DISCUSSION TITLE
Program wide darkmode

### DISCUSSION THREAD

**Discussion by AustinMarstaller** [fjwillemsen/NativeOverleaf#11] [Ideas]
Dear all,

I have just begun using NativeOverleaf primarily for the Dark mode feature. It is fine when you are working within a specific project, but when navigating back to your projects, the dark mode specification is gone and replaced with the default light mode. It would be nice to enforce "dark mode" in the entire program instead of just within the projects.

**Comment by fjwillemsen** [comment]
Hi @AustinMarstaller! I've just released v1.7.6, which extends the dark mode to other panels and the project overview. Let me know what you think!
